src/solver.jl

```diff
@@ -415,18 +194,9 @@ function gondzio_correction_step!(solver, correction_lb, correction_ub, ind_cons
 end
 
 # Predictor-corrector method
-function mpc!(solver::MadNLP.AbstractMadNLPSolver; max_ncorr=0)
```

Comment:
I suppose that `max_ncorr` was for the multiple corrections variant of Gondzio but not implemented?

Reply:
`max_ncorr` has been moved in the structure `IPMOptions`. Gondzio's multiple corrections have been implemented already :)